src/pineko/check.py

```diff
@@ -41,3 +42,30 @@ def check_grid_and_eko_compatible(pineappl_grid, operators):
     # x-grid
     if not np.all(in1d(np.unique(operators["targetgrid"]), np.array(x_grid))):
         raise ValueError("x grid in pineappl grid and eko operator are NOT compatible!")
+
+
+def check_grid_contains_sv(grid_path, xir, xif, ftr):
+    """Raises a `ValueError if the theory_card asks for scale-variations but they are not
+    available in the pineappl grid.
+
+    Parameters
+    ----------
+        grid_path : pathlib.Path
+            path to grid
+        xir : float
+            log of renormalization scale ratio to central
+        xif : float
+            log of factorization scale ratio to central (scheme-C)
+        ftr : float
+            log of factorization scale ratio to central (scheme-B)
+    """
+    if xir == 1 and xif == 1 and ftr == 1:
```

Comment:
the function should take a single argument, the grid and this if-condition goes in the calling function in the theory module

Reply:
also the check needs to use `isclose` (float remember ...)